src/store/modules/nft.js

```diff
@@ -441,6 +474,7 @@ const actions = {
             catchAxiosError(dispatch('fetchNFTPurchaseInfo', classId))
           );
         }
+        promises.push(dispatch('fetchNFTPurchaseInfo', classId));
```

Comment:
Can this part be simplified?

Reply:
You mean put this to line 465?

Reply:
Yes, will the logic make sense?

Reply:
Actually this should be `fetchNFTListingInfo`